tests/wicked/before_test.pm

```diff
@@ -31,7 +31,9 @@ sub run {
     assert_script_run('sed -e "s/^WICKED_DEBUG=.*/WICKED_DEBUG=\"all\"/g" -i /etc/sysconfig/network/config');
     assert_script_run('sed -e "s/^WICKED_LOG_LEVEL=.*/WICKED_LOG_LEVEL=\"debug\"/g" -i /etc/sysconfig/network/config');
     assert_script_run('cat /etc/sysconfig/network/config');
-    assert_script_run('mkdir /data');
+    #preparing directories for holding config files
+    assert_script_run('mkdir -p /data/static_address');
```

Comment:
if we want to have single line for directories we could call mkdir -p /data/{static_address,dynamic_address}

Reply:
Done